test/NotifierTest.cpp

```diff
@@ -209,3 +428,74 @@ TEST_F(NotifierTest, 2Messages2Receivers_RestartReceivers) {
    EXPECT_TRUE(ThreadIsShutdown(receiver2ThreadData));
    EXPECT_TRUE(ThreadIsShutdown(senderData));
 }
+
+TEST_F(NotifierTest, 2Messages2Receivers_RestartReceivers_VectorMessage) {
```

Comment:
to add  to test name ```__expected_Feedback``` or similar (do the same for the one that also has feedback)